PET_Clinic/PET_BUSINESS_TRANSACTIONS.md

```diff
@@ -0,0 +1,72 @@
+
+### Creating Business Transaction Definitions
+
+In the next step, we can group the data of an application and combine them with a certain business context, for example: the business transaction *Owner* contains all data belonging to the process of editing the details of an owner.
+
+#### 1. Creating Business Transaction for Owner
+
+First, switch to the **Business Transaction Definitions** tab (at the bottom of the view) and click on the **Create** button. As before, a dialog appears where we can name the newly created business transaction definition. Enter a name, e.g. *Owner* and click on the *Finish* button.
+
+Like a application definition, a business transaction definition consists of **business transaction mappings** which are used to correlate data to a business transaction.
+
+As before, create an **URI Matcher** which accepts all data where the URI starts with the value ```/api/customer/owners```. Press ```CTRL + S``` to save the changes. And that's it.
+
+#### 2. Creating Business Transaction for Static Resources
+
+Second, we would like to see all requests that are related to static resources. Create a new business tracsaction with the name "Static Resources". As before, create an **URI Matcher** which accepts all data where the URI matches the regex ```.*(js|css|woff|woff2|png)```.  
+
+#### 3. Dynamic Business Transaction Extraction
+
+Third, we still have requests that are not mapped to a Business Transaction. Thus, we would like to dynamically extract the name of the business transaction depending on the **URI**. As before, create a new Business Trancation named *Dynamic BT* and add a **URI Matcher** which accepts all data where the URI starts with the value ```/```. Enable the "Dynamic Name Extraction" on the right side. Activate the checkbox "Extract name dynamically" and select the source based on which the name should be extracted. Choose **HTTP URI** from the dropdown list. In the **Name Pattern Specification** select **Specify name pattern** and add:
+```
+Regular Expressions: /([^/]*).*
+Name Pattern: BT - (1)
+```
+
+This way the first part of the URI will be extracted and used as business transaction name, i.e. /owner/details --> "BT - owner" OR /pet/visit --> "BT - pet".
+
+#### 4. Advanced Rules for Creating Business Transactions
+In the Business Transaction Mapping multiple rules can be used to define one business transaction. These rules are linked with OR conditions. In case AND conditions should be used enable the "advanced rule view". In our case we want to split the business transaction *Owner* further into  "Edit Owner" and "Get All Owner" business transactions. 
```

Comment:
space too much between into "Edit Owner"

Would be nice to explain why you are doing this. You can explain that two BT are using same URi, thus we want to split them up.